imgee/storage.py

```diff
@@ -275,8 +282,12 @@ def clean_local_cache(expiry=24):
 def delete(stored_file, commit=True):
     """
     Delete all the thumbnails and images associated with a file, from local cache and S3.
-    Wait for the upload/resize to complete if queued for the same image.
     """
+    registry = imgee.registry
+
+    # remove from registry
+    registry.remove_keys_starting_with(stored_file.name)
```

Comment:
oh, I'm sorry. it removes all keys matching `imgee:registry:default:<name>*.` 

File names are `<name>.<extn>` for original file and `<name>_wNN_hNN.<extn>` for thumbnails. the redis keys dont have extensions. just the `<name>` part. so they are like `imgee:registry:default:<name>` or `imgee:registry:default:<name>_wNN_hNN`, We're deleting keys starting with `<name>` here.

in `__init__()` function, 
```
def init_app(self, name='default', connection=None):
        self.connection = connection or self.set_connection_from_url()
        self.name = name
        self.key_prefix = 'imgee:registry:%s' % name
        ...
```
The `<name>` in that key_prefix is the name of the registry, which is set to `default`. i'm not sure why that name is there.